be/src/olap/types.h

```diff
@@ -255,6 +255,32 @@ struct BaseFieldtypeTraits : public CppTypeTraits<field_type> {
     }
 };
 
+template <typename T>
+OLAPStatus convert_int_from_varchar(void* dest, const void* src) {
+    using SrcType = typename CppTypeTraits<OLAP_FIELD_TYPE_VARCHAR>::CppType;
+    auto src_value = *reinterpret_cast<const SrcType*>(src);
+    StringParser::ParseResult parse_res;
+    T result = StringParser::string_to_int<T>(src_value.get_data(), src_value.get_size(), &parse_res);
+    if (UNLIKELY(parse_res != StringParser::PARSE_SUCCESS)) {
+        return OLAPStatus::OLAP_ERR_INVALID_SCHEMA;
+    }
+    *reinterpret_cast<T*>(dest) = result;
```

Comment:
for largeint value, because dest may be not align of 16 bytes, this operation may cause process crash.
So better to use memcpy for largeint